### PR TITLE
SCAL-96721: Remove/Replace iframe instances in the param description …

### PR DESCRIPTION
**Pull request by aditya-kumar44**
…text blocks and Remove @hidden markers for QueryChanged and AuthExpire

### CODE REVIEW

File: src/embed/ts-embed.ts [hunk 1]
```diff
@@ -587,6 +587,10 @@ export class TsEmbed {
         return this;
     }
 
+    /**
+     * @method
+     * @ignore
```
Comment (jbhanu-thoughtspot) [inline src/embed/ts-embed.ts:592]: Let's instead evaluate and remove the use of this method `test_setIframe`

File: src/embed/ts-embed.ts [hunk 2]
```diff
@@ -521,7 +521,7 @@ export class TsEmbed {
     /**
      * Navigate to particular page. eg:answers/pinboards/home
      * This is used for embedding answers, pinboards, visualizations and full application.
-     * @param path The string, set to iframe src and navigate to new page
+     * @param path The string, set to enclosed viewport src and navigate to new page
```
Comment (jbhanu-thoughtspot) [inline src/embed/ts-embed.ts:524]: Correct the statements

File: src/types.ts [hunk 3]
```diff
@@ -232,7 +232,7 @@ export enum EmbedEvent {
      */
     AuthInit = 'authInit',
     /**
-     * The iFrame has loaded. This only refers to the iFrame load event
+     * The enclosed viewport has loaded. This only refers to the enclosed viewport load event
```
Comment (jbhanu-thoughtspot) [inline src/types.ts:235]: Get this relooked with Shashi

File: src/types.ts [hunk 4]
```diff
@@ -291,8 +289,8 @@ export enum EmbedEvent {
      */
     EmbedHeight = 'EMBED_HEIGHT',
     /**
-     * The center of visible iframe viewport is calculated.
```
Comment (jbhanu-thoughtspot) [inline src/types.ts:294]: Add @hidden tag for internal events

File: src/types.ts [hunk 5]
```diff
@@ -291,8 +289,8 @@ export enum EmbedEvent {
      */
     EmbedHeight = 'EMBED_HEIGHT',
```
Comment (kailash-choudhary-thoughtspot) [inline src/types.ts:290]: hide this event

File: src/types.ts [hunk 6]
```diff
@@ -291,8 +289,8 @@ export enum EmbedEvent {
...
     /**
-     * The center of visible iframe viewport is calculated.
-     * @return data - The center of the visible Iframe viewport.
+     * The center of visible enclosed viewport is calculated.
+     * @return data - The center of the visible enclosed viewport.
      */
     EmbedIframeCenter = 'EmbedIframeCenter',
```
Comment (kailash-choudhary-thoughtspot) [inline src/types.ts:295]: hide this event

**Pull request closed without merging**